erpnext-fhir-mapper/src/main/java/com/ozonehis/eip/model/erpnext/QuotationItem.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.model.erpnext;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class QuotationItem implements ERPNextDocument {
```

Comment:
It might be necessary here to override `equals()` and `hashCode()` either via Lombok or manually. The version added by `@Data` includes all properties, but the assumptions of `Quotation` are that `customExternalID` uniquely identifies each `QuotationItem`